deploy-board/deploy_board/templates/deploys/deploy_details.tmpl

```diff
@@ -83,6 +83,12 @@
             {% include "builds/build_details.tmpl" %}
         </td>
     </tr>
+    <tr>
+        <td>Account</td>
+        <td>
+            <a href="/clouds/accounts/{{ account_info.cloudProvider }}/{{ account_info.cell }}/{{ account_info.id }}">{{ account_info.id }}</a>
```

Comment:
Let's hide this link if account is `None` for backward compatibility in cases when Deploy Board will be deployed before cluster management service.

Or if deployment was done in the cluster which was provisioned manually.

Reply:
updated, thanks